python/mlir/tpp/sched/common.py

```diff
@@ -1,12 +1,17 @@
 from mlir.dialects import transform
-from mlir.dialects.transform import structured
+from mlir.dialects.transform import structured, tune
 
 
 # Wrapper to addresss verbosity.
 def apply_registered_pass(*args, **kwargs):
     return transform.apply_registered_pass(transform.AnyOpType.get(), *args, **kwargs)
 
 
+# Wrapper to addresss verbosity.
+def select(*args, **kwargs):
+    return tune.select(transform.AnyParamType.get(), *args, **kwargs)
```

Comment:
Unrelated to the PR itself but what's the point of `op_name` in `transform.select`?
AFAIK, usually I'd just use the SSA result and pass that forward. Can I somehow refer to the name from a different (transform) op?